src/snyk/common/constants/commands.ts

```diff
@@ -17,6 +17,8 @@ export const SNYK_OPEN_ISSUE_COMMAND = 'snyk.showissue';
 export const SNYK_IGNORE_ISSUE_COMMAND = 'snyk.ignoreissue';
 export const SNYK_SHOW_OUTPUT_COMMAND = 'snyk.showOutputChannel';
 export const SNYK_SHOW_LS_OUTPUT_COMMAND = 'snyk.showLsOutputChannel';
+export const SNYK_GET_LESSON_COMMAND = 'snyk.getLearnLesson';
+export const SNYK_CLI_CONFIG_SAST_ENABLED = 'snyk.cliConfigSettings.sastEnabled';
```

Comment:
Why not to name command consistently with others in LS as a verb, e.g. `get...`?